docs/src/pages/docs/functions/combinators.md

```diff
@@ -20,6 +20,84 @@ give it a value and it will give you back a function ready to take a function.
 Once that function is provided, it will return the result of applying your value
 to that function.
 
+```javascript
+import applyTo from 'crocks/combinators/applyTo'
+
+import compose from 'crocks/helpers/compose'
+import First from 'crocks/First'
+import flip from 'crocks/combinators/flip'
+import isArray from 'crocks/predicates/isArray'
+import isNumber from 'crocks/predicates/isNumber'
+import isString from 'crocks/predicates/isString'
+import map from 'crocks/pointfree/map'
+import merge from 'crocks/pointfree/merge'
+import mreduceMap from 'crocks/helpers/mreduceMap'
+import Pair from 'crocks/Pair'
+import safeLift from 'crocks/Maybe/safeLift'
+
+// prices :: [ Number ]
```

Comment:
What do you think about `Array Number` instead of `[ Number ]`?

Reply:
Well, if it's convention then I'd be happy to change it. But since we're so free-handed with Haskell signatures, and especially given that it's a number declaration, I think it's fine.

Reply:
I think this is one that is inconsistent within the docs, that's why I asked. i think `Array Number` is more consistent with other type containing signatures, however, `[ Number ]` has a nicer JS feel to it

Reply:
Yeah, it's not quite clear if Array is supposed to be the convention here, given that this prefers `[]` notation. I'd be happy with either.
https://github.com/evilsoft/crocks/blame/master/docs/src/pages/docs/crocks/Arrow.md#L233